nibabel/volumeutils.py

```diff
@@ -504,7 +507,8 @@ def array_from_file(shape, in_dtype, infile, offset=0, order='F', mmap=True):
             pass
     if len(shape) == 0:
         return np.array([])
-    n_bytes = int(np.prod(shape) * in_dtype.itemsize)
```

Comment:
You could also specify the `dtype` in `np.prod`, rather than introduce `operator.mul`:

    n_bytes = int(np.prod(shape, dtype=np.int64) * in_dtype.itemsize)